src/main/python/ayab/scene.py

```diff
@@ -124,14 +124,20 @@ def refresh(self):
                 -5.5,
                 0,
                 pixmap.height() + 5.5))
+
+        # Draw knitting progress
         qscene.addItem(
             QGraphicsRectItem(
-                self.__start_needle - machine_width / 2 - 1,
-                pixmap.height() + 0.5,
-                self.__stop_needle - self.__start_needle + 3,
+                - machine_width / 2 - 1,
```

Comment:
This looks suspicious - is self.__start_needle missing here?
Please verify @t0mpr1c3 , then I'll do the merge. Thanks!